translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3319 @@
+msgid ""
+"The name of the API generator to use. There are 2 generators: 'service/v1' "
+"and 'service/v2'. The only difference between them is that service port in "
+"v1 is named 'default', while it is left unnamed in v2. Default is 'service/"
+"v2'."
+msgstr ""
+"O nome do gerador de API a ser usado. Existem 2 geradores: 'service/v1' e "
+"'service/v2'. A única diferença entre eles é que a porta de serviço na v1 é "
+"chamada de 'default', enquanto ela é deixada sem nome na v2. O padrão é "
+"'service/v2'."
+
+#: pkg/kubectl/cmd/run.go:136
+msgid ""
+"The name of the generator to use for creating a service.  Only used if --"
+"expose is true"
+msgstr ""
+"O nome do generator para ser utilizado quando criando um serviço.  Apenas "
+"utilizado se —expose é verdadeiro"
+
+#: pkg/kubectl/cmd/expose.go:100
+msgid "The network protocol for the service to be created. Default is 'TCP'."
+msgstr "O protocolo de rede para o serviço ser criado. Padrão é 'TCP'."
+
+#: pkg/kubectl/cmd/expose.go:101
+msgid ""
+"The port that the service should serve on. Copied from the resource being "
+"exposed, if unspecified"
+msgstr ""
+"A porta para que o serviço possa servir. Copiado do recurso sendo exposto, "
+"se não especificado"
+
+#: pkg/kubectl/cmd/run.go:124
+msgid ""
+"The port that this container exposes.  If --expose is true, this is also the "
+"port used by the service that is created."
+msgstr ""
+"A porta que o container expõe.  Se —expose é verdadeiro, esta também é a "
+"porta utilizada pelo serviço quando for criado."
+
+#: pkg/kubectl/cmd/run.go:134
+msgid ""
+"The resource requirement limits for this container.  For example, 'cpu=200m,"
+"memory=512Mi'.  Note that server side components may assign limits depending "
+"on the server configuration, such as limit ranges."
+msgstr ""
+"O recurso requerido para este container.  Por exemplo, 'cpu=200m,"
+"memory=512Mi'.  Observe que os componentes do lado do servidor podem "
+"atribuir limites, dependendo da configuração do servidor, como intervalos de "
+"limite."
+
+#: pkg/kubectl/cmd/run.go:133
+msgid ""
+"The resource requirement requests for this container.  For example, "
+"'cpu=100m,memory=256Mi'.  Note that server side components may assign "
+"requests depending on the server configuration, such as limit ranges."
+msgstr ""
+"O recurso requerido de requests para este container.  Por exemplo, 'cpu=100m,"
+"memory=256Mi'.  Observe que os componentes do lado do servidor podem "
+"atribuir requests, dependendo da configuração do servidor, como intervalos "
+"de limite."
+
+#: pkg/kubectl/cmd/run.go:131
+msgid ""
+"The restart policy for this Pod.  Legal values [Always, OnFailure, Never].  "
+"If set to 'Always' a deployment is created, if set to 'OnFailure' a job is "
+"created, if set to 'Never', a regular pod is created. For the latter two --"
+"replicas must be 1.  Default 'Always', for CronJobs `Never`."
+msgstr ""
+"A politica de restart para este Pod.  Possíveis valores [Always, OnFailure, "
```

Comment:
Probably this needs some update as kubectl run now creates a Pod and the restart policy is related to this